test/verify/check-machines-dbus

```diff
@@ -292,6 +295,60 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_in_text("#vm-subVmTest1-network-1-source", "test_network")
         b.wait_not_present("#vm-subVmTest1-network-1-source-tooltip")
 
+        # Remove the network interface
+        m.execute("virsh detach-interface --persistent --type network --domain subVmTest1")
+
+        # We don't get events for shut off VMs so reload the page
+        b.reload()
+        b.enter_page('/machines')
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_in_text(".cards-pf #card-pf-networks .card-pf-title-link", "Networks")
+        b.wait_in_text("#card-pf-networks .card-pf-aggregate-status-count", "2")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+        b.click("tbody tr[data-row-id=vm-subVmTest1] th") # click on the row header
+        b.click("#vm-subVmTest1-networks") # open the "Networks" subtab
+
+        # Change network type and source from the bridge NIC
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "bridge")
+        b.wait_in_text("#vm-subVmTest1-network-1-source", "virbr0")
+
+        # Open the modal dialog
+        b.click("#vm-subVmTest1-network-1-edit-dialog")
+
+        b.wait_in_text("#vm-subVmTest1-network-1-select-type", "Bridge to LAN")
+        b.select_from_dropdown("#vm-subVmTest1-network-1-select-type", "Virtual network")
+        b.select_from_dropdown("#vm-subVmTest1-network-1-select-source", "test_network")
+
+        # Save the network settings
+        b.click("#vm-subVmTest1-network-1-edit-dialog-save")
+        b.wait_not_present("#vm-subVmTest1-network-1-edit-dialog-modal-window")
+
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "network")
+        b.wait_in_text("#vm-subVmTest1-network-1-source", "test_network")
+
+        # bridge-utils are not available on the following test images
+        if m.image not in ["rhel-8-0-distropkg", "rhel-8-0", "rhel-8-1", "debian-testing"]:
+            # Remove all Virtual Networks and confirm that trying to choose
+            # Virtual Networks type for a NIC disables the save button
+            m.execute("virsh net-destroy test_network && virsh net-destroy default && virsh net-undefine test_network && virsh net-undefine default")
+
+            # Remove the network interface
+            m.execute("virsh detach-interface --persistent --type network --domain subVmTest1")
+
+            # Start the VM in order that the UI picks up the new interface, since we don't get events of shut off domains
+            m.execute("virsh start subVmTest1")
+
+            # Create a second bridge to LAN NIC
+            m.execute("brctl addbr br1 && virsh attach-interface --current subVmTest1 bridge br1")
```

Comment:
As we've seen in #11840, let's not use the obsolete brctl here, but the `bridge` command.

Reply:
Sure, I was not aware that it's not used any-more.
Replaced the command in the tests. Sorry for the confusion.